### PR TITLE
Add support for GraphQL 16 Peer Dependency

### PR DESCRIPTION
**Pull request by drew-y**
We've been using this package with GraphQL 16 without issue.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,90 +1,127 @@
 # Changelog
+
 ## [Unreleased]
 
+## [1.6.1] - 2020-07-03
```
Comment (stylesuxx) [inline CHANGELOG.md:5]: Please put your changes in the unreleased section, I will version and date it once I publish it.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,90 +1,127 @@
...
+
+- Updated graphql peer dependency to ^16
+
 ## [1.6.0] - 2020-07-03
+
```
Comment (stylesuxx) [inline CHANGELOG.md:10]: Could you please remove all the empty lines.

**Review by stylesuxx — CHANGES_REQUESTED** [review]
Awesome, thank you for your contribution. Could you please just fix the two small changes then I will merge and publish a new version.

File: CHANGELOG.md (hunk 2 above)
Comment (drew-y) [reply CHANGELOG.md:10]: Yep! Sorry about that!

Comment (drew-y) [reply CHANGELOG.md]: Done.

File: CHANGELOG.md (hunk 1 above)
Comment (drew-y) [reply CHANGELOG.md]: Yep! No problem. Moved.

**Review by stylesuxx — APPROVED** [review]

**Comment by stylesuxx** [comment]
Thank you, merged and published with 1.7.0

**Pull request merged**